docs/docs/Concepts/RAFS.md

```diff
@@ -139,6 +139,91 @@ If there are multiple copies of the same chunk referenced by a RAFS filesystem,
 For more information, please refer to [Chunk Deduplication](./chunk-dedup.md)
 
 ### Data Prefetch
+The data prefetch works together with data cache，which makes plaintext representations cached in the local disk. When IO requests hit cache, the RAFS does not need to pull data from the storage backend, thus reducing latency caused by lazy loading.
```

Comment:
How about: `where prefetched data will be stored on local disk in uncompressed or compressed format`.